### PR TITLE
Remove html elements from ScubaResults.json

### LINKED ISSUE #1290: Remove HTML elements from ScubaResults.json
# 💡 Summary #
https://github.com/cisagov/ScubaGear/pull/1281 adds a new function, Format-PlainText that removes the HTML elements from the control statements / report details that don't make sense to include in a csv file. Call this same function (or a similar one) to remove elements that don't make sense to include in the json. 

## Motivation and context ##
Not all elements prepared for the HTML reports make sense to include in the json version. For example, the "View all CA policies" links in the AAD report don't make sense to include in the json report as that link is specific to the AAD HTML report and is broken out of context.

## Implementation notes ##
Recommend starting with the Format-PlainText function, tweak or make a new one as needed.

## Acceptance criteria ##
- [ ] The HTML elements ScubaGear add to the HTML report for the baseline statements have been replaced
- [ ] The HTML elements ScubaGear add to the HTML report for the report details have been replaced

### PR DESCRIPTION
**Pull request by ehaines1**
Removed html tags from ScubaResults.json

## 🗣 Description ##

I used the exisiting Format-PlainText method to strip out html tags in ScubaResults.json, specifically for links, policies and report details.  This was already being done for ScubaResults.csv. 

## 💭 Motivation and context ##

Closes #1290 

## 🧪 Testing ##

To test it, run ScubaGear and verify that the "links" render as text without unicode tags.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] PR targets the correct parent branch (e.g., main or release-name) for merge.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] Changes are sized such that they do not touch excessive number of files.
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] These code changes follow the ScubaGear [content style guide](https://github.com/cisagov/ScubaGear/blob/main/CONTENTSTYLEGUIDE.md).
- [x] Related issues these changes resolve are linked preferably via [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] All relevant type-of-change labels added.
- [x] All relevant project fields are set.
- [x] All relevant repo and/or project documentation updated to reflect these changes.
- [x] Unit tests added/updated to cover PowerShell and Rego changes.
- [x] Functional tests added/updated to cover PowerShell and Rego changes.
- [x] All relevant functional tests passed.
- [x] All automated checks (e.g., linting, static analysis, unit/smoke tests) passed.

## ✅ Pre-merge checklist ##

- [x] PR passed smoke test check.
- [x] Feature branch has been rebased against changes from parent branch, as needed

  Use `Rebase branch` button below or use [this](https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request) reference to rebase from the command line.
- [x] Resolved all merge conflicts on branch
- [x] Notified merge coordinator that PR is ready for merge via comment mention

## ✅ Post-merge checklist ##

- [x] Feature branch deleted after merge to clean up repository.
- [ ] Verified that all checks pass on parent branch (e.g., main or release-name) after merge.

### CODE REVIEW

**Comment by ehaines1** [comment]
Some functional tests have failed, but didn't seem to be related to the changes I made

File: PowerShell/ScubaGear/Modules/Orchestrator.psm1 [hunk 1]
```diff
@@ -1046,14 +1049,22 @@ function Merge-JsonOutput {
...
+                        $Control.Requirement = Format-PlainText -RawString $Control.Requirement
+                        $Control.Details = Format-PlainText -RawString $Control.Details
+                    }
+                }
+            }
 
             # Convert the output a json string
             $MetaData = ConvertTo-Json $MetaData -Depth 3
             $Results = ConvertTo-Json $Results -Depth 5
+           # Loop through each property
```
Comment (adhilto) [inline PowerShell/ScubaGear/Modules/Orchestrator.psm1:1067]: A very very minor thing...I don't understand this comment. There's no looping here. Maybe a copy-paste mistake? Or maybe I'm missing something obvious.

Comment (ehaines1) [reply PowerShell/ScubaGear/Modules/Orchestrator.psm1:1067]: I took out the comment.  I think I had started doing something and then changed my approach so it's no longer relevant

**Review by adhilto — APPROVED** [review]

**Comment by ehaines1** [comment]
@nanda-katikaneni I think this is ready to merge!  There was one sharepoint functional test run that failed, but I checked with Addam and he confirmed that this is the error we're figuring out with Microsoft.  Otherwise, everything seemed to pass

**Pull request merged**